const createErroror = require("../utils/createerroror");
exports.createJoblist = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
exports.updateJoblist = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
exports.deleteJoblist = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
